extensions/amp-ad/0.1/amp-ad-cross-domain-iframe-handler.js

```diff
@@ -107,6 +106,9 @@ export class AmpAdApiHandler {
       // If support render-start, create a race between render-start no-content
       this.adResponsePromise_ = listenForOncePromise(this.iframe_,
         ['render-start', 'no-content'], this.is3p_).then(info => {
+          if (this.timeout_) {
```

Comment:
I removed `this.timeout_`. Only one case that we can possibly call `noContent` twice is `bootstrap-loaded` never arrives, but `no-content` arrives. 
I think this is fine because`integration.js` sends the `bootstrap-loaded`and it should always arrives. Even it never arrives,  `noContent_` guard multiple calls by checking `this.iframe_`.